src/meta/src/barrier/command.rs

```diff
@@ -84,6 +84,13 @@ impl Command {
         Self::Plain(Some(Mutation::Resume(ResumeMutation {})))
     }
 
+    pub fn changed_actor_ids(&self) -> Option<Vec<ActorId>> {
```

Comment:
Do agree with this function for scaling, which is very similar to why `changed_table` is necessary. However, according to the design of dropping materialized views, barriers in the actors to stop should still be collected. So this should only include the newly created actors instead, since they are marked inactive and won't be selected by default.

Besides, how about making this more general for configuration change instead of naming it `migration` here?